### ISSUE TITLE
INSTITUTE: "Top" button does not return user to the top

### ISSUE THREAD

**Issue by aaronMidd** [middlebury/midd-frontend#400]
On the new Institute site - scrolling to a certain point (about 50%) triggers the display of a "Top" button in the lower right corner. 

Clicking this SHOULD return the user to the top.

It does nothing.

**Comment by imcbride** [comment]
The Olark button is on top of it preventing the click action from working.

**Comment by aaronMidd** [comment]
Yup - will you look at that. It is working on other pages. Nice sleuthing.

**Comment by imcbride** [comment]
Temporary fix is in place which restores the functionality of the button.
